doc/conf.py

```diff
@@ -141,8 +142,13 @@ def _check_deps():
 master_doc = 'contents'
 
 # General substitutions.
-from subprocess import check_output
-SHA = check_output(['git', 'describe', '--dirty']).decode('utf-8').strip()
+try:
+    SHA = subprocess.check_output(
+        ['git', 'describe', '--dirty']).decode('utf-8').strip()
+# Catch the case where git is not installed locally, and use the versioneer
+# version number instead
+except subprocess.CalledProcessError:
```

Comment:
actually that raises a filenotfounderror, not a calledprocesserror, when git is not in the $PATH